tox.ini

```diff
@@ -32,7 +34,7 @@ commands =
 [testenv:docs]
 passenv = *
 deps=
-    Sphinx==1.4.4
+    Sphinx
     sphinx_bootstrap_theme
```

Comment:
Same thing for the docs. If I want to build the documentation I have to read the tox file instead of just installing requirements-*.txt files.